templates/account/confirm_login_code.html

```
@@ -1,48 +1,50 @@
-      <input type="hidden" name="next" value="/accounts/login" />
-      {% csrf_token %}
-    </form>
+{% block auth_body %}
+{% include "account/snippets/auth_form_errors.html" with form=form %}
+{% url 'account_confirm_login_code' as login_url %}
+<form class="flex w-full flex-col gap-6" method="POST" action="{{ login_url }}">
+  {% csrf_token %}
+  <div class="space-y-4 px-1 py-4">
+    <div class="auth-field flex h-[84px] w-full flex-col justify-center gap-2 overflow-hidden border-b border-[var(--auth-field-border)] py-[14px] pr-[22px] text-left text-base leading-6 text-text-strong">
+      <label class="m-0" for="id_code">{% translate "Code" %}</label>
+      <input class="m-0 block h-6 w-full border-0 bg-transparent p-0 leading-6 text-text-strong outline-none placeholder:text-[var(--auth-text-muted)] focus-visible:ring-0" required name="code" placeholder="{% translate 'Code' %}" id="id_code" autocomplete="one-time-code" type="text"{% if form.code.errors %} aria-invalid="true" aria-describedby="id_code-errors"{% endif %}>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)